src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -47,6 +55,19 @@ internal static IntellisenseQuickInfoItem BuildItem(ITrackingSpan trackingSpan,
                 quickInfoItem.Sections.Where(s => s.Kind != QuickInfoSectionKinds.Description)
                                       .Select(BuildClassifiedTextElement));
 
+            // build text for RelatedSpan
+            if (quickInfoItem.RelatedSpans.Any())
+            {
+                var classifiedSpanList = new List<ClassifiedSpan>();
+                foreach (var span in quickInfoItem.RelatedSpans)
+                {
+                    var classifiedSpans = await Classifier.GetClassifiedSpansAsync(document, span, cancellationToken).ConfigureAwait(false);
```

Comment:
Note: this function is not safe to call for a non-C#/VB language.

Reply:
in the short term, i would use this.  but just remove the contract assert.  in the medium term, i would use the method GetClassifiedSpansAsync in ClassifiedSpansAndHighlightSpanFactory.  However, i'd like to make a few tweaks to that method first to make it more suitable.  I'll open a PR for this.

Reply:
@JieCarolHu The PR is: https://github.com/dotnet/roslyn/pull/28152